rep-2000.rst

```diff
@@ -1,20 +1,69 @@
 REP: 2000
-Title: ROS 2 Target Platforms
-Author: Mikael Arguedas <mikael@openrobotics.org>, Steven! Ragnarok <steven@openrobotics.org>
+Title: ROS 2 Releases and Target Platforms
+Author: Mikael Arguedas <mikael@openrobotics.org>, Steven! Ragnarok <steven@openrobotics.org>, Dirk Thomas <dthomas@openrobotics.org>
 Status: Active
 Type: Informational
 Content-Type: text/x-rst
-Created: 10-Apr-2018, 24-Apr-2018, 21-May-2018, 10-Dec-2018, 20-May-2019, 21-May-2019, 29-Aug-2019
+Created: 10-Apr-2018, 24-Apr-2018, 21-May-2018, 10-Dec-2018, 20-May-2019, 21-May-2019, 29-Aug-2019, 25-Feb-2020
 
 
 Abstract
 ========
 
-This REP defines target platforms for each ROS 2 Distribution Release.
+This REP defines timeline for future ROS 2 releases as well as the targeted platforms for each specific one.
```

Comment:
Plural 'timelines' to match 'releases'

```suggestion
This REP defines timelines for future ROS 2 releases as well as the targeted platforms for each specific one.
```

Reply:
Thanks, I inserted a "the" since I think there is only one:

> This REP defines the timeline for future ROS 2 releases